drivers/can/can_xlnx.c

```diff
@@ -0,0 +1,1329 @@
+/*
+ * Xilinx Processor System CAN controller driver
+ *
+ * Copyright (c) 2022, Weidmueller Interface GmbH & Co. KG
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Known limitations / TODOs:
+ * - Sleep mode / PM functionality not considered at the time being.
+ * - High Priority TX buffer not considered at the time being.
```

Comment:
Please leave out TODOs from the code. From the code it can easily be seen that PM functionality is not implemented - and the high-priority TX buffer does not fit into the Zephyr CAN API and likely never will.